joern-cli/src/main/resources/scripts/c/malloc-overflow.sc

```diff
@@ -1,18 +1,21 @@
-import io.shiftleft.codepropertygraph.Cpg
-import io.shiftleft.codepropertygraph.generated.Operators
 import io.shiftleft.codepropertygraph.generated.nodes.Call
-import io.shiftleft.semanticcpg.language._
-import overflowdb.traversal._
 
-@main def main(): List[Call] = {
-  cpg
+@main def main(inputPath: String) = {
+  importCode(inputPath)
+  val calls = cpg
     .call("malloc")
     .filter { mallocCall =>
       mallocCall.argument(1) match {
         case subCall: Call =>
           subCall.name == Operators.addition || subCall.name == Operators.multiplication
         case _ => false
       }
-    }
-    .l
+    }.code
+
+  val expected = Set(
+    "malloc(sizeof(int) * 42)",
+    "malloc(sizeof(int) * 3)",
+    "malloc(sizeof(int) + 55)"
+  )
+  assertContains("calls", calls, expected)
```

Comment:
Did this turn into some kind of test? Before we did not assert anything. Did the use of this script change. Some question applies to the other `.sc` files.

Reply:
I just noticed that those expectation came from `RunTestScripts.scala`. Why did you change the setup? Having the tests in the actual implementation seems strange.